src/Agent.Worker/Release/ContainerFetchEngine/IConatinerFetchEngineLogger.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Microsoft.VisualStudio.Services.Agent.Worker.Release.ContainerFetchEngine
+{
+    [ServiceLocator(Default = typeof(NullExecutionLogger))]
+    // NOTE: FetchEngine specific interface shouldn't take dependency on Agent code.
```

Comment:
It's a good constraint to have if we were to move the download functionality into a separate EXE. However let's target this goal subsequently, not needed right now.